Makefile

```diff
@@ -46,22 +46,19 @@ endif
 
 GO_PROJECT=github.com/ceph/ceph-csi
 
+CEPH_VERSION ?= $(shell . $(CURDIR)/build.env ; echo $${CEPH_VERSION})
+# TODO: ceph_preview tag may be removed with go-ceph 0.16.0
+GO_TAGS_LIST ?= $(CEPH_VERSION) ceph_preview
```

Comment:
using preview API by default was not accepted see https://github.com/ceph/ceph-csi/pull/2633#discussion_r751977715 we need a stable API as this will be used in production?

Reply:
Here's a list of preview go-ceph APIs that this PR relies on:
* `rados.ReadOp.AssertVersion()`
* `rados.ReadOp.GetOmapValuesByKeys()` and its return value `rados.ReadOpOmapGetValsByKeysStep struct`
* `rados.ReadOp.Read()`
* `rados.WriteOp.AssertVersion()`
* `rados.WriteOp.Remove()`
* `rados.WriteOp.SetXattr()`

These are just bare bindings for librados for functions that are stable -- not really any brand new functionality that would be in a big risk of breaking in following two go-ceph releases (the time needed to graduate an API from preview to stable). Is this not acceptable too?

Reply:
So how do we do this, can anybody please let me know? Can I continue prototyping other parts of #2142 while this is in review, or are you postponing this PR until `ceph_preview` build tag is no longer needed?

All of these bindings used in this PR are for RADOS primitives, which are very unlikely to change.

Reply:
am fine with this one as it was not accepted in earlier PR i mentioned it here. @humblec @nixpanic?

Reply:
fine with me too. :+1: